modules/server-node/src/index.ts

```diff
@@ -1340,59 +1340,60 @@ server.post<{ Body: NodeParams.SendExitChannelTx }>(
-          request.body,
-        ).toJson(),
-      );
-  }
-
-  let record: StoredTransaction | undefined;
-  try {
-    record = await store.getTransactionByHash(request.body.transactionHash);
-  } catch (e) {
-    return reply.status(500).send(
-      new ServerNodeError(ServerNodeError.reasons.StoreMethodFailed, request.body.publicIdentifier, request.body, {
-        storeError: e.message,
-        storeMethod: "getTransactionByHash",
-      }).toJson(),
-    );
-  }
-
-  if (!record) {
-    return reply
-      .status(404)
-      .send(
-        new ServerNodeError(
-          ServerNodeError.reasons.TransactionNotFound,
-          request.body.publicIdentifier,
-          request.body,
-        ).toJson(),
-      );
-  }
-  const result = await chainService.speedUpTx(record.chainId, {
-    to: record.to,
-    data: record.data,
-    value: record.value,
-    nonce: record.nonce,
-    transactionHash: record.transactionHash,
-  });
-  if (result.isError) {
-    return reply.status(500).send(jsonifyError(result.getError()!));
-  }
-  return reply.status(200).send({ transactionHash: result.getValue().transactionHash });
-});
+// TODO: does speed up make sense still?
+// server.post<{ Body: NodeParams.SpeedUpTx }>("/speed-up", async (request, reply) => {
+//   if (request.body.adminToken !== config.adminToken) {
+//     return reply.status(401).send({ message: "Unauthorized" });
+//   }
+
+//   const chainService = getChainService(request.body.publicIdentifier);
+//   if (!chainService) {
+//     return reply
+//       .status(404)
+//       .send(
+//         new ServerNodeError(
+//           ServerNodeError.reasons.ChainServiceNotFound,
+//           request.body.publicIdentifier,
+//           request.body,
+//         ).toJson(),
+//       );
+//   }
+
+//   let record: StoredTransaction | undefined;
+//   try {
+//     record = await store.getTransactionByHash(request.body.transactionHash);
+//   } catch (e) {
+//     return reply.status(500).send(
+//       new ServerNodeError(ServerNodeError.reasons.StoreMethodFailed, request.body.publicIdentifier, request.body, {
+//         storeError: e.message,
+//         storeMethod: "getTransactionByHash",
+//       }).toJson(),
+//     );
+//   }
+
+//   if (!record) {
+//     return reply
+//       .status(404)
+//       .send(
+//         new ServerNodeError(
+//           ServerNodeError.reasons.TransactionNotFound,
+//           request.body.publicIdentifier,
+//           request.body,
+//         ).toJson(),
+//       );
+//   }
+//   const result = await chainService.speedUpTx(record.chainId, {
+//     to: record.to,
+//     data: record.data,
+//     value: record.value,
+//     nonce: record.nonce,
+//     transactionHash: record.transactionHash,
+//   });
+//   if (result.isError) {
+//     return reply.status(500).send(jsonifyError(result.getError()!));
+//   }
+//   return reply.status(200).send({ transactionHash: result.getValue().transactionHash });
+// });
```

Comment:
I was thinking the same thing about speed up - imo, makes more sense to obfuscate / deprecate this functionality for now, relying instead on our own internal 'speeding up' logic. There may be individual use cases down the road where someone might like to use the speed up functionality, so we'll leave room to re-implement, but I agree with this move.

Left a note on ethService - do we want to remove that method for `speedUpTx` as well?